File: java/Paypal.java
public class Paypal extends Payment {

    String email;
    String password;

    public Paypal(Integer id, Integer monto, String email, String password){
        super(id, monto);
        this.email = email;
        this.password = password;
    }

    void emailPrint(){
        System.out.println("email:" + email);
    }
    
}

File: js/Paypal.js
class Paypal extends Payment{

    constructor(id, monto, email, password){

        super(id, monto)
        this.email = email
        this.password = password

    }

    printDataPaypal(){
        console.log("email: " + this.email + " password: " + this.password );
    }

}